Makefile

```diff
@@ -173,6 +173,18 @@ gofmt: ## Runs gofmt against all packages.
 	done
 	@echo "gofmt success"; \
 
+golangci-lint:
+# https://stackoverflow.com/a/677212/1027058 (check if a command exists or not)
+# https://github.com/golangci/golangci-lint#binary-release
+# It is recommended to NOT use go get, but instead use a binary release pinned to a version.
+	@if ! [ -x "$$(command -v golangci-lint)" ]; then \
+		echo "Installing golangci-lint"; \
+		curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(GOPATH)/bin v1.21.0; \
```

Comment:
I don't think we should install software so easily. I think we should just report, that golangci-lint is needed.

Reply:
Well, it was needed for the CI to run. :) But now if we are using the SAS service, we don't need this.